website/content/en/docs/olm-integration/olm-deployment.md

```diff
@@ -0,0 +1,144 @@
+      --manifests string            [olm only] Directory containing package manifest and operator bundles.
+      --operator-version string     [olm only] Version of operator to deploy
+      --install-mode string         [olm only] InstallMode to create OperatorGroup with. Format: InstallModeType=[ns1,ns2[, ...]]
+      --include strings             [olm only] Path to Kubernetes resource manifests, ex. Role, Subscription. These supplement or override defaults generated by run/cleanup
+      --timeout duration            [olm only] Time to wait for the command to complete before failing (default 2m0s)
+      ...
+```
+
+These flags correspond to options in the `OLMCmd` configuration model (names are similar if not identical):
+
+```go
+type OLMCmd struct {
+  // KubeconfigPath is the local path to a kubeconfig. This uses well-defined
+  // default loading rules to load the config if empty.
+  KubeconfigPath string
+  // OLMNamespace is the namespace in which OLM is installed.
+  OLMNamespace string
+  // OperatorNamespace is the cluster namespace in which operator resources
+  // are created.
+  // OperatorNamespace must already exist in the cluster or be defined in
+  // a manifest passed to IncludePaths.
+  OperatorNamespace string
+  // ManifestsDir is a directory containing a package manifest and N bundles
+  // of the operator's CSV and CRD's. OperatorVersion can be set to the
+  // version of the desired operator version's subdir and Run()/Cleanup() will
+  // deploy the operator version in that subdir.
+  ManifestsDir string
+  // OperatorVersion is the version of the operator to deploy. It must be
+  // a semantic version, ex. 0.1.0.
+  OperatorVersion string
+  // InstallMode specifies which supported installMode should be used to
+  // create an OperatorGroup. The format for this field is as follows:
+  //
+  // "InstallModeType=[ns1,ns2[, ...]]"
+  //
+  // The InstallModeType string passed must be marked as "supported" in the
+  // CSV being installed. The namespaces passed must exist or be created by
+  // passing a Namespace manifest to IncludePaths. An empty set of namespaces
+  // can be used for AllNamespaces.
+  // The default mode is OwnNamespace, which uses OperatorNamespace or the
+  // kubeconfig default.
+  InstallMode string
+  // IncludePaths are path to manifests of Kubernetes resources that either
+  // supplement or override defaults generated by methods of OLMCmd. These
+  // manifests can be but are not limited to: RBAC, Subscriptions,
+  // CatalogSources, OperatorGroups.
+  //
+  // Kinds that are overridden if supplied:
+  // - CatalogSource
+  // - Subscription
+  // - OperatorGroup
+  IncludePaths []string
+  // Timeout dictates how long to wait for a REST call to complete. A call
+  // exceeding Timeout will generate an error.
+  Timeout time.Duration
+}
+```
```

Comment:
Probably don't want Go code snippets in user-facing docs when the docs aren't about Go-specific topics.